src/manual.c

```diff
@@ -186,8 +186,14 @@ set_initial_history(char *name)
 		pathFile = popen(buf, "r");
 		if (fgets(buf, sizeof(buf), pathFile)==NULL)
 		{
-			fprintf(stderr, "Error executing command '%s'\n", buf);
-			exit(1);
+			/* Try without -W */
```

Comment:
This lacks `pclose(pathFile);` to not leak that `FILE`.

Reply:
Added.